deploy/mapping/encode.mcf

```diff
@@ -0,0 +1,33 @@
+Node: E:EncodeBedLine->E1
```

Comment:
I think we should add the BedLine mappings to base.mcf, and use that for encode KG? 

Since the tables are all the same.

Reply:
This is included in main and encode kg. Note there are 3 mapping files base.mcf, weather.mcf and encode.mcf. Separate them out so it's easy to maintain.

Reply:
Ah, got it...